### DISCUSSION TITLE
CSS + Animations in Svelte

### DISCUSSION THREAD

**Discussion by jonas-dongowski** [themeselection/sneat-bootstrap-html-admin-template-free#15] [General]
Hey there!

I'd love to use this template in my Svelte project.
The CSS already works like a charm, now the collapse for the mobile menu is the only problem I face.
Can you give me a quick summary of how exactly the menu works so I can write my own Svelte version of it?
Bonus would be getting the collapse on larger screens to work but I don't necessarily need that.

I'm down to buy the pro version once I get this to work properly as I am a big fan of your dark theme.

I hope we can get this done.

Yours
Jonas

**Comment by saadaamir3047** [comment]
Hey Jonas,

I'm not the developer for this, but I can give you a brief overview of how the menu works. The menu uses CSS classes and JavaScript to create a collapsing effect. The CSS initially hides the menu items, and when a specific button is clicked, JavaScript triggers a class toggle on the menu container to show or hide the items. To create your own Svelte version, set up your project, import the CSS, create a component, add HTML structure, and use Svelte's event handling to toggle the menu's visibility. You can also modify the JavaScript to make the collapse work on larger screens. Good luck with your project and enjoy the dark theme!

Best,
Saad
